src/Stripe/Services/Subscriptions/StripeSubscriptionCreateOptions.cs

```diff
@@ -42,6 +42,24 @@ internal string TrialEndInternal
         [JsonProperty("tax_percent")]
         public decimal? TaxPercent { get; set; }
 
+        public DateTime? BillingCycleAnchor { get; set; }
+
+        public bool BillingCycleAnchorNow { get; set; }
+
+        [JsonProperty("billing_cycle_anchor")]
+        internal string BillingCycleAnchorInternal
```

Comment:
Same thing, also Internal Stripe documentation. But this production ready and supported by Stripe, but they haven't made this public yet. Also another library in PHP supports this: https://github.com/laravel/cashier/pull/114